src/components/Calendar/CalendarDate/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import RawButton, { Props as RawButtonProps } from '../../RawButton';
+import { ymdToDateString } from '../../../utils';
+
+import styles from './styles.css';
+
+export interface Props {
+    className?: string;
+    year: number;
+    month: number;
+    date: number;
```

Comment:
Shouldn't this be day?
```suggestion
    day: number;
```

Reply:
day represents day of week in js